docs/docs/workflows/s3proxy.md

```diff
@@ -20,12 +20,12 @@ If you want to use s3proxy but these limitations stop you from doing so, conside
 You can add the s3proxy to your Constellation cluster as follows:
 1. Download the deployment manifest:
    ```bash
-   wget https://raw.githubusercontent.com/edgelesssys/constellation/main/s3proxy/deploy/deployment-s3proxy.yaml
+   git clone --depth=1 git@github.com:edgelesssys/constellation.git
    ```
-2. Replace the values named `replaceme` in `deployment-s3proxy.yaml` with valid AWS credentials. These credentials are used by s3proxy to access your S3 buckets.
+2. Set AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY to valid credentials you want s3proxy to use to interact with S3. Set IMAGE to the [s3proxy docker image](https://github.com/edgelesssys/constellation/pkgs/container/constellation%2Fs3proxy) you want to use.
```

Comment:
```suggestion
2. Set AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY to valid credentials you want s3proxy to use to interact with S3.
```
nit: We should just make sure the image in `values.yaml` is set to a valid value